### PR TITLE
addressing breaking changes to pagination introduced by rb

### PR DESCRIPTION
**Pull request by Javran**
now using react-ultimate-pagination to wrap around rb's own Pagination.

removed compatibility code from `onSelectPage` of `views/browse-area/index.es` since it is rb's own breaking change in the first place.

plan to bump to 3.9.0 after this due to having quite some features implemented already.

### CODE REVIEW

File: views/browse-area/sortie-viewer/index.es [hunk 1]
```diff
@@ -378,19 +377,16 @@ class SortieViewerImpl extends PureComponent {
...
-              next
-              first
-              last
-              ellipsis
-              boundaryLinks
-              maxButtons={5}
-              onSelect={this.handleSelectPage}
-            />
+            {
+              (activePage <= pageRange) && (
```
Comment (yukixz) [inline views/browse-area/sortie-viewer/index.es]: 这是啥0 0

Comment (Javran) [reply views/browse-area/sortie-viewer/index.es]: 现在rb的Pagination把一些逻辑去掉了, 然后官方推荐做法是用react-ultimate-pagination之类的东西套一层来处理,这地方的UPagination就是套好的.

https://www.npmjs.com/package/@react-bootstrap/pagination 照这个抄的, 只是加了个把style传给wrapper component 的处理.

Comment (yukixz) [reply views/browse-area/sortie-viewer/index.es]: emm... 我的意思是 `activePage <= pageRange` 这个

Comment (Javran) [reply views/browse-area/sortie-viewer/index.es]: 哦. 不加那部分会报错, pageRange初始值是0 (还没读完的时候), 而activePage最小是1. 大概是ultimate-pagination自己带的检测机制.

**Pull request merged**